docs/excel/custom-functions-errors.md

```diff
@@ -33,15 +30,17 @@ function getCity(zipCode: string): string {
 
 ## The CustomFunctions.Error object
 
-The `CustomFunctions.Error` object is used to return an error back to the cell. When you create the object, specify which error you want to use by using one of the following `ErrorCode` enum values.
+The `CustomFunctions.Error` object is used to return an error back to the cell. When you create the object, specify which error you want to use by choosing one of the following `ErrorCode` enum values.
 
 
 |ErrorCode enum value  |Excel cell value  |Meaning  |
 |---------------|---------|---------|
-|`invalidValue`   | `#VALUE!` | A value used in the formula is the wrong type. |
-|`notAvailable`   | `#N/A`    | The function or service isn't available. |
 |`divisionByZero` | `#DIV/0`  | Be aware that JavaScript allows division by zero so you need to write an error handler carefully to detect this condition. |
-|`invalidNumber`  | `#NUM!`   | There is a problem with the number used in the formula |
+|`invalidName`    | `#NAME?`  | There is a typo in the function name. Note: this error is supported as a custom function input error, but not as a custom function output error. | 
```

Comment:
Instead of the "Note: ..." structure, consider phrasing this like "Note that ...". It avoids the awkward colon.
```suggestion
|`invalidName`    | `#NAME?`  | There is a typo in the function name. Note that this error is supported as a custom function input error, but not as a custom function output error. | 
```